helper/dockertest/network.go

```diff
@@ -0,0 +1,35 @@
+package dockertest
+
+import (
+	"context"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	dockhelper "github.com/hashicorp/vault/sdk/helper/docker"
+)
+
+func CreateNetwork(t *testing.T, name string) func() {
```

Comment:
Is this unique for our testing or can this be imported from the Vault repo?

Reply:
Removed this logic. Now we use the actual IP address and don't create networks.